ios/BetterSocial.xcodeproj/project.pbxproj

```diff
@@ -8,7 +8,7 @@
 
 /* Begin PBXBuildFile section */
 		00E356F31AD99517003FC87E /* BetterSocialTests.m in Sources */ = {isa = PBXBuildFile; fileRef = 00E356F21AD99517003FC87E /* BetterSocialTests.m */; };
-		0E0E0A4F064AF84E69E81068 /* libPods-OneSignalNotificationServiceExtension.a in Frameworks */ = {isa = PBXBuildFile; fileRef = D1D792BA444DA06DB9C6F91A /* libPods-OneSignalNotificationServiceExtension.a */; };
+		036832A7C6B969325240D615 /* libPods-OneSignalNotificationServiceExtension.a in Frameworks */ = {isa = PBXBuildFile; fileRef = D1D792BA444DA06DB9C6F91A /* libPods-OneSignalNotificationServiceExtension.a */; };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The reference to the library `libPods-OneSignalNotificationServiceExtension.a` has been changed. If this was intentional and the new library is compatible with the rest of the code, then it's fine. However, if this change was not intended, it could potentially lead to issues in the functionality related to OneSignal notifications.

```diff
- 0E0E0A4F064AF84E69E81068 /* libPods-OneSignalNotificationServiceExtension.a in Frameworks */ = {isa = PBXBuildFile; fileRef = D1D792BA444DA06DB9C6F91A /* libPods-OneSignalNotificationServiceExtension.a */; };
+ 036832A7C6B969325240D615 /* libPods-OneSignalNotificationServiceExtension.a in Frameworks */ = {isa = PBXBuildFile; fileRef = D1D792BA444DA06DB9C6F91A /* libPods-OneSignalNotificationServiceExtension.a */; };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->